metrics/web/test/features/common/presentation/drawer/widget/metrics_drawer_test.dart

```diff
@@ -24,10 +24,10 @@ void main() {
   );
 }
 
-class MetricsDrawerTestbed extends StatelessWidget {
+class _MetricsDrawerTestbed extends StatelessWidget {
```

Comment:
Should we add documentation for this testbed? And maybe this is a stub - I am not sure of this when we're talking about widgets.